docs/CHANGELOG.rst

```diff
@@ -2,6 +2,10 @@
 Changelog
 =========
 
+* Fix isort dependency
+* Added tests for Python 3.10
+* Introduce new setup.cfg and pyproject.toml
+* Update __authors__
```

Comment:
Where are the new authors?

Reply:
There are no new, but I just changed the list in the __init__.py file